libcxx/test/std/utilities/meta/meta.unary/meta.unary.prop/reference_constructs_from_temporary.pass.cpp

```diff
@@ -9,7 +9,7 @@
 // REQUIRES: std-at-least-c++23
 
 // These compilers don't support std::reference_converts_from_temporary yet.
-// UNSUPPORTED: android, apple-clang-15, apple-clang-16, clang-19.1
+// UNSUPPORTED: android, clang-19.1
```

Comment:
@arichardson What would be a proper versioned identifier we can use here instead of just `android`? Otherwise this test is going to be disabled forever on Android regardless of the compiler in use.